doc/source/whatsnew/v0.15.0.rst

```diff
@@ -318,8 +318,8 @@ Timezone handling improvements
      ts
      ts.tz_localize(None)
 
-     didx = pd.DatetimeIndex(start='2014-08-01 09:00', freq='H',
-                             periods=10, tz='US/Eastern')
+     didx = pd.date_range(start='2014-08-01 09:00', freq='H',
```

Comment:
can you remove the ":okwarning:" a few lines above?

Reply:
okwarning removed + green